bot/admin/admin.py

```diff
@@ -133,6 +133,85 @@ async def embed_available_cogs(self, ctx: commands.Context):
 
         await ch_bot.send(embed=embed)
 
+    @cmd_for_bot_stuff.command(name='load', hidden=True)
+    @command_log
+    async def load_extension(self, ctx: commands.Context, extn_name: str):
+        """Command handler for the `bot` subcommand `load`.
+
+        Loads an extension (Cog) with the specified name into the bot.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            extn_name (str): The name of the extension (Cog).
+        """
+        extn_name = extn_name.capitalize()
+
+        if "Cog" and "cog" not in extn_name:
+            extn_name += "Cog"
+        elif "cog" in extn_name:
+            extn_name = extn_name[:-3] + "Cog"
+
+        self.bot.load_extension(constants.INITIAL_EXTNS[extn_name])
+        print(f"\t* {extn_name} has been loaded.")
+
+    @cmd_for_bot_stuff.command(name='unload', hidden=True)
+    @command_log
+    async def unload_extension(self, ctx: commands.Context, extn_name: str):
+        """Command handler for the `bot` subcommand `unload`.
+
+        Removes an extension (Cog) with the specified name from the bot.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            extn_name (str): The name of the extension (Cog).
+        """
+        extn_name = extn_name.capitalize()
+
+        if "Cog" and "cog" not in extn_name:
```

Comment:
Same as above in load function. This does not work and is highly unstable

Reply:
Also if you had this code where you replace the name in a separate function, not only could you test it beforehand with a test, you could also reuse it and not duplicate the same code.